llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -769,10 +769,10 @@ class LoadInst final : public Instruction {
   unsigned getNumOfIRInstrs() const final { return 1u; }
   static LoadInst *create(Type *Ty, Value *Ptr, MaybeAlign Align,
                           Instruction *InsertBefore, Context &Ctx,
-                          const Twine &Name = "");
+                          bool isVolatile = false, const Twine &Name = "");
```

Comment:
@vporpo I got it work, I was having signature issues with the compiler when trying to do 4 functions.